InterviewView.swift

```diff
@@ -0,0 +1,39 @@
+
+import UIKit
+
+class ViewController: UIViewController {
+    var output: ProfileViewOutput?
+    let resultButton = UILabel()
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        resultButton.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor).isActive = true
```

Comment:
you need to add subview before setting constraints, also you need to change translatesAutoresizingMaskIntoConstraints field to false for setting constraints in code